andmarti1424/sc-im
# basic digraphs for issue #356

## A simple implementation of Vim-like digraphs

### How to test
Bring up a spreadsheet. Add or edit a cell label. Entering `Ctrl-k` will pause the cursor. Enter `p` then `*` - the letter `&pi;`. Should also work for replacing a character in a cell label.

### Caveat
You can also use the digraphs in filenames.
